pytest_operator/plugin.py

```diff
@@ -77,6 +77,13 @@ def pytest_addoption(parser):
         help="Store the completed crash dump in this dir. "
         "The default is current folder.",
     )
+    parser.addoption(
+        "--model-config",
+        action="store",
+        default="model_config.yaml",
```

Comment:
I would default this to `None` or `""`. Better not to accidentally load some file lying around; the user should have to pass it in explicitly.